admin-dev/themes/new-theme/js/pages/order/create/product-renderer.js

```diff
@@ -454,4 +461,22 @@ export default class ProductRenderer {
   _hideNotFound() {
     $(createOrderMap.noProductsFoundWarning).addClass('d-none');
   }
+
+  /**
+   * Shows searching product notice
+   *
+   * @private
+   */
+  _showSearching() {
```

Comment:
```suggestion
  showSearching() {
```
I think @PierreRambaud will tell you not tu use `_` prefix and it will be removed when merged into `develop` because ESLint will refuse it

Reply:
I know I anticipated this kind off feedback 😁 It just feels weird to have a mix in 177.. some start by _ some not..
But I totally understand how it will ease the merge in develop so who cares 😅